doc/user/quick-sql-grammar.md

```diff
@@ -79,7 +79,6 @@
 | /compress, /compressed                  | Table will be created compressed. |
 | /insert NN                              | Generate NN SQL INSERT statement(s) with random data, for example: /INSERT 20. (Maximum = 1000) |
 | /rest                                   | Generate REST enablement of the table using Oracle REST Data Services (ORDS) |
-| /select                                 | Generate SQL SELECT statement after generating data for each table |
```

Comment:
why was select nuked @vadim-tropashko ?

Reply:
Legacy syntax, not implemented. Asked on forum, and nobody replied.